src/Owin.Security.OpenIdConnect.Extensions/OpenIdConnectConstants.cs

```diff
@@ -110,6 +124,8 @@ public static class Metadata {
             public const string SubjectTypesSupported = "subject_types_supported";
             public const string TokenEndpoint = "token_endpoint";
             public const string UserinfoEndpoint = "userinfo_endpoint";
+            //Added by Introspection (but not in Spec?)
+            public const string IntrospectionEndpoint = "introspection_endpoint";
```

Comment:
It's indeed not a standardized thing, but I agree it would be useful to share it via the configuration endpoint. If you keep it, don't forget to update the configuration endpoint to expose it.